TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/chassis/ChassisConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package org.firstinspires.ftc.teamcode.subsystems.chassis;
+
+import com.acmerobotics.dashboard.config.Config;
+import com.arcrobotics.ftclib.geometry.Translation2d;
+
+@Config
+public interface ChassisConfiguration {
+    String MOTOR_LEFT_FRONT = "MotorLeftFront";
+    String MOTOR_RIGHT_FRONT = "MotorRightFront";
+    String MOTOR_LEFT_BACK = "MotorLeftBack";
+    String MOTOR_RIGHT_BACK = "MotorRightBack";
+
+    /**
+     * Locations are relative to the center of the robot
+     */
+    Translation2d frontLeftLocation = new Translation2d(0, 0);
+    Translation2d frontRightLocation = new Translation2d(0, 0);
+    Translation2d backLeftLocation = new Translation2d(0, 0);
+    Translation2d backRightLocation = new Translation2d(0, 0);
```

Comment:
why is this needed exactly? we aren't doing any fancy corrections for steering or anything. I don't think ftclib is needed in this case anyway, since all we are trying to do is have the most barebones chassis for teleop (all manual controls), so I think that just purely adding speeds up correctly is all that's needed here.

Maybe this has some sort of other use that is relevant for us though i do not think so

Reply:
ftclib has a really nice `MecanumDriveOdometry` utility, which depends on having a configured MecanumDriveKinematics object

I agree, for teleop it does seem unecessary but I thought it'd be nice to be able to use the same chassis subsystem for autonomous as well

Reply:
autonomous will be using roadrunners mecanum chassis impementation (it is required for pedropathing), and without a lot of work we won't be able to get this to work

Reply:
maybe we'd be able go use the roadrunners chassis for teleop, but i think its overkill and too cluttered. we need a simple inputs->wheel speeds class, no more no less